src/components/AddItem/AddItem.component.jsx

```diff
@@ -57,13 +50,7 @@ const AddItem = props => {
         const items = snapshot.docs
 
           .map(query => query.data())
-
-          .map(data =>
```

Comment:
Please see the above comment with the corrected code block for lines 62-66.

``` javascript
          .map(data =>
            data.name
              .toLowerCase()
              .replace(/[^\w\s]|/g, '')
              .replace(/[\s]/, ''),
          );
```

Reply:
Fixed